pkg/api/v1/clients/kube/resource_client_test.go

```diff
@@ -57,16 +57,17 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	kubeClient, err := kubernetes.NewForConfig(cfg)
 	Expect(err).NotTo(HaveOccurred())
 
-	lock, err = clusterlock.NewTestClusterLocker(kubeClient, clusterlock.Options{
+	lock, err = clusterlock.NewKubeClusterLocker(kubeClient, clusterlock.Options{
```

Comment:
Just moved away from the deprecated API. This is the method the deprecated API was calling